CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+## General contribution guidelines
+
+* Have a look at open issues. Pick an unassigned issue. If doubtful, discuss on the issue itself.
+* You can also create a new issue for a feature or bug.
+* Make sure you are assigned to the issue before you start working on it.
+* Make sure you are not working on more than 2 issues at once.
+* While sending a PR make sure you follow one issue per PR rule.
+* Squash your commits in all PRs you send.
+* Make sure the Issue and PR template are correctly filled.
+* Make sure to write good commit messages as explained [here](https://hackernoon.com/what-makes-a-good-commit-message-995d23687ad).
```

Comment:
Check this out https://docs.taskcluster.net/manual/design/devel/best-practices/commits. Instead of adding the link writing something of that sort in Contribution.md itself may be better

Reply:
Done.